Modules/_decimal/docstrings.h

```diff
@@ -399,72 +1066,198 @@ PyDoc_STRVAR(doc_radix,
 "radix($self, /)\n--\n\n\
 Return Decimal(10), the radix (base) in which the Decimal class does\n\
 all its arithmetic. Included for compatibility with the specification.\n\
+\n\
+    >>> ExtendedContext.radix()\n\
+    Decimal('10')\n\
 \n");
 
 PyDoc_STRVAR(doc_remainder_near,
 "remainder_near($self, /, other, context=None)\n--\n\n\
-Return the remainder from dividing self by other.  This differs from\n\
-self % other in that the sign of the remainder is chosen so as to minimize\n\
-its absolute value. More precisely, the return value is self - n * other\n\
-where n is the integer nearest to the exact value of self / other, and\n\
-if two integers are equally near then the even one is chosen.\n\
+Returns to be \"a - b * n\", where n is the integer nearest the exact\n\
```

Comment:
This sentence reads a bit off-kilter to me:

```suggestion
Return to be \"a - b * n\", where n is the integer nearest the exact\n\
```